migrations/0053_domain_isced_migration.py

```diff
@@ -0,0 +1,118 @@
+# Generated by Django 2.2.10 on 2020-04-23 15:52
+import csv
+from pathlib import Path
+
+from django.conf import settings
+from django.db import migrations, models
+from django.db.models import Exists, F, OuterRef, Subquery
+
+
+def forward(apps, schema_editor):
+    PartnershipYear = apps.get_model('partnership', 'PartnershipYear')
+    DomainIsced = apps.get_model('reference', 'DomainIsced')
+    PartnershipYearEducationField = apps.get_model(
+        'partnership', 'PartnershipYearEducationField'
+    )
+
+    # Get unmatched education fields codes
+    missing_codes = PartnershipYearEducationField.objects.annotate(
+        matching=Exists(DomainIsced.objects.filter(
+            code=OuterRef('code')
+        ))
+    ).filter(matching=False).values_list('code', flat=True)
+
+    # Load CSV into a mapping by code
+    mapping = {}
+    with Path(settings.BASE_DIR, 'partnership/fixtures/domains_isced.csv').open(encoding='utf8') as f:
+        csv_content = csv.DictReader(f, delimiter=";")
+        for row in csv_content:
+            mapping[row['code']] = {**row, 'is_ares': bool(row['is_ares'])}
+
+    new_objects = []
+    for missing_code in missing_codes:
+        new_objects.append(DomainIsced(**mapping[missing_code]))
```

Comment:
Ici est-ce que je dois garder l'uuid du `PartnershipYearEducationField` pour le nouveau `DomainIsced` ?

Reply:
@SebCorbin Tout dépend si tu l'utilises dans le contexte de l'API?

Reply:
Oui, je l'utilise mais l'API renvoyant les education_field est elle-même modifiée, donc la question se résume à : est-ce qu'on veut que les liens avec filtres tel que https://partnership-test.osis.uclouvain.be/#/partners?education_field=1ce448e6-ae4a-4092-913f-07e49103cf9e&ordering=country_en,city restent valides après cette migration

Reply:
@SebCorbin , non ce n'est pas nécessaire. 
Les valeurs sont utilisées uniquement dans le cadre d'une lecture
Merci pour la précision